.github/workflows/ci-windows.yml

```diff
@@ -62,6 +62,12 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: ./aom.cmd
+      # Visual Studio 2022 has an issue starting at 17.8.0 which might cause
+      # AVX-512 instructions to be emitted in non-AVX-512 code. See
+      # https://github.com/AOMediaCodec/libavif/issues/2033#issuecomment-1960062751.
+      env:
```

Comment:
order doesn't matter, but to me `run:` makes sense to be last, since the other fields affect it.